src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/CA/EligibilityCheckSpec.groovy

```diff
@@ -151,4 +162,51 @@ class EligibilityCheckSpec extends GebReportingSpec {
         excludedReasonsItem(3).checked
         excludedReasonsItem(5).checked
     }
+
+    def 'The CRD time form is shown after the suitability form' () {
+        given: 'On the eligibility checks page'
+        at EligibilityCheckPage
+
+        when: 'I select new options and save'
+        excludedRadios.checked = 'Yes'
+        find('#continueBtn').click()
+
+        then: 'I see the suitability form'
+        at EligibilityCheckSuitablePage
+
+        when: 'I select yes for unsuitable and continue'
+        unsuitableRadios.checked = 'Yes'
+        find('#continueBtn').click()
+
+        then: 'I see the crd time form'
+        at EligibilityCheckCrdTimePage
+    }
+
+    def 'All selections are saved and shown on the task list' () {
+        given: 'On the eligibility checks page'
+        actions.toEligibilityCheckPageFor('A1235HG')
+        at EligibilityCheckPage
+
+        when: 'I select new options and save'
+        excludedRadios.checked = 'Yes'
+        find('#continueBtn').click()
+        at EligibilityCheckSuitablePage
+
+        and: 'I select No for unsuitable and continue'
+        unsuitableRadios.checked = 'No'
+        find('#continueBtn').click()
+        at EligibilityCheckCrdTimePage
+
+        and: 'I select Yes for CRD time and continue'
+        crdTimeRadios.checked = 'Yes'
+        find('#continueBtn').click()
+
+        then: 'I return to the task list page'
+        at TaskListPage
+
+        and: 'I can see my saved answers'
+        excludedAnswer.text() == 'Yes'
+        unsuitableAnswer.text() == 'No'
+        crdTimeAnswer.text() == 'Yes'
+    }
```

Comment:
I realise that this is testing behaviour of a different page but it seemed a sensible way of testing that it was saved. Let me know if you have a better solution